pkg/proxy/conntrack/conntrack.go

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
I'd say put this in `pkg/proxy/util/conntrack` too. No reason to have two separate conntrack directories.
You can call this file "cleanup.go" or something like that.